src/commands/bitcount.json

```diff
@@ -54,24 +58,31 @@
                         "type": "integer"
                     },
                     {
-                        "name": "end",
-                        "type": "integer"
-                    },
-                    {
-                        "name": "unit",
-                        "type": "oneof",
+                        "name": "end-unit-block",
+                        "type": "block",
                         "optional": true,
-                        "since": "7.0.0",
                         "arguments": [
                             {
-                                "name": "byte",
-                                "type": "pure-token",
-                                "token": "BYTE"
+                                "name": "end",
+                                "type": "integer"
```

Comment:
```suggestion
                                "type": "integer",
                                "optional": true,,
                                "since": "8.0.0"
```

Reply:
Also, the one I am wondering..
argument `end` was existed before, and this change is just making it as optional. `since` is also needed in this case?